### PR TITLE
Vxstream do not detonate files in submit-url

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/26767

## Description
Added the `dontThrowErrorOnFileDetonation` argument in **crowdstrike-submit-url*

## Screenshots
image

## Does it break backward compatibility?
   - [ ] Yes
       - Further details: 
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CrowdStrikeFalconSandbox/Integrations/integration-CrowdStrikeFalconSandbox.yml [hunk 1]
```diff
@@ -66,6 +66,22 @@ script:
...
 
+    // Add `contains` to String prototype.
+
+    if (!('contains' in String.prototype))
+        String.prototype.contains = function(str, startIndex) {
+            return -1 !== String.prototype.indexOf.call(this, str, startIndex);
+        };
+
+    function argToBool(arg){
+        switch (typeof arg){
```
Comment (Itay4) [inline Packs/CrowdStrikeFalconSandbox/Integrations/integration-CrowdStrikeFalconSandbox.yml:77]: not sure i understand why is that needed, it will always be string

File: Packs/CrowdStrikeFalconSandbox/Integrations/integration-CrowdStrikeFalconSandbox.yml [hunk 2]
```diff
@@ -606,11 +622,20 @@ script:
...
         if (version === 'v1') {
             throw 'This command is supported only in API v2.'
         }
-        var response = submitUrl(url, environmentID);
+        try {
+          var response = submitUrl(url, environmentID);
+        } catch (exception) {
+              var notSupported = 'The provided URL resolves to a file.'
+              if (dontThrowErrorOnFileDetonation && exception.contains(notSupported)){
+                  return notSupported;
```
Comment (Itay4) [inline Packs/CrowdStrikeFalconSandbox/Integrations/integration-CrowdStrikeFalconSandbox.yml:634]: lets return a better error message here that the file type is not supported

File: Packs/CrowdStrikeFalconSandbox/Playbooks/playbook-Detonate_URL_-_CrowdStrike.yml [hunk 3]
```diff
@@ -151,8 +167,10 @@ tasks:
       brand: VxStream
     nexttasks:
       '#none#':
-      - "16"
+      - "17"
     scriptarguments:
+      dontThrowErrorOnFileDetonation:
```
Comment (Itay4) [inline Packs/CrowdStrikeFalconSandbox/Playbooks/playbook-Detonate_URL_-_CrowdStrike.yml:172]: this should be an input to the playbook with `false` as default

File: Packs/CrowdStrikeFalconSandbox/Playbooks/playbook-Detonate_URL_-_CrowdStrike.yml [hunk 4]
```diff
@@ -247,22 +275,67 @@ tasks:
...
+    skipunavailable: false
+    quietmode: 0
+  "17":
+    id: "17"
+    taskid: 063f8872-26bf-4bf9-8caa-645a82166659
+    type: condition
+    task:
+      id: 063f8872-26bf-4bf9-8caa-645a82166659
+      version: -1
+      name: Is url detonated?
```
Comment (Itay4) [inline Packs/CrowdStrikeFalconSandbox/Playbooks/playbook-Detonate_URL_-_CrowdStrike.yml:298]: why is this needed?

**Review by Itay4 — CHANGES_REQUESTED** [review]
please revert the readme name refactor and add the new arg to it

File: Packs/CrowdStrikeFalconSandbox/Playbooks/playbook-Detonate_URL_-_CrowdStrike.yml (hunk 4 above)
Comment (jochman) [reply Packs/CrowdStrikeFalconSandbox/Playbooks/playbook-Detonate_URL_-_CrowdStrike.yml:298]: If the url did not detonated - Skip to the done task and not fail the playbook

File: Packs/CrowdStrikeFalconSandbox/Integrations/integration-CrowdStrikeFalconSandbox.yml (hunk 2 above)
Comment (jochman) [reply Packs/CrowdStrikeFalconSandbox/Integrations/integration-CrowdStrikeFalconSandbox.yml:634]: It does not support any files. Just webpages.

File: Packs/CrowdStrikeFalconSandbox/Playbooks/playbook-Detonate_URL_-_CrowdStrike.yml (hunk 3 above)
Comment (jochman) [reply Packs/CrowdStrikeFalconSandbox/Playbooks/playbook-Detonate_URL_-_CrowdStrike.yml:172]: the command default is false

File: Packs/CrowdStrikeFalconSandbox/Integrations/integration-CrowdStrikeFalconSandbox.yml (hunk 2 above)
Comment (Itay4) [reply Packs/CrowdStrikeFalconSandbox/Integrations/integration-CrowdStrikeFalconSandbox.yml:634]: change to something like `The file format is not supported, use <COMMAND NAME> instead.`

File: Packs/CrowdStrikeFalconSandbox/Playbooks/playbook-Detonate_URL_-_CrowdStrike.yml (hunk 4 above)
Comment (Itay4) [reply Packs/CrowdStrikeFalconSandbox/Playbooks/playbook-Detonate_URL_-_CrowdStrike.yml:298]: change to `Was job created ? (could be it wasn't created because file type is not supported)`

File: Packs/CrowdStrikeFalconSandbox/Integrations/integration-CrowdStrikeFalconSandbox.yml [hunk 5]
```diff
@@ -606,11 +622,20 @@ script:
...
         if (version === 'v1') {
             throw 'This command is supported only in API v2.'
         }
-        var response = submitUrl(url, environmentID);
+        try {
+          var response = submitUrl(url, environmentID);
+        } catch (exception) {
+              var notSupported = 'The provided URL resolves to a file.'
+              if (dontThrowErrorOnFileDetonation && exception.contains(notSupported)){
+                  return 'The file format is not supported, use "crowdstrike-submit-file-by-url" instead.';
```
Comment (Itay4) [inline Packs/CrowdStrikeFalconSandbox/Integrations/integration-CrowdStrikeFalconSandbox.yml:634]:
```suggestion
                  return 'The file format is not supported, use the command "crowdstrike-submit-file-by-url" instead.';
```

**Review by Itay4 — APPROVED** [review]
looks good 
please note my suggestion above
